docs/token-contracts/fa12/2-fa12-ligo.md

```diff
@@ -53,10 +53,10 @@ To deploy your asset contract;
 1. Choose the "Deploy" action under the "Configure" pick-list.
 1. In the "Storage" field, you shall see the initial data for this Smart
    Contract. You must update two items:
-       * Replace `YOUR_TZ_ADDRESS` with the address you copied from your faucet
-     file earlier
-       * Replace *both* occurrences of `TOKENS` with the total number of tokens
-     you wish to issue.
+   _ Replace `YOUR_TZ_ADDRESS` with the address you copied from your faucet
```

Comment:
not sure what this leading underscore here is supposed to mean, looks like broken formatting when rendered